dpc-sdp/ripple-framework
# [R20-1321] Add custom collection

**Issue**: https://digital-vic.atlassian.net/browse/R20-1321

### What I did

- Added TideCustomCollection component - this will be used for custom search experiences in Landing page content types. It will be used first for the Cameras save lives location table.
- Adds API endpoint for proxying elasticsearch requests seperate to app search (would be good to also consolidate content collection in a follow up PR)
- Refactors search page components to be consistent

### How to test

- Added integration tests
- To test manually
- - Set `NUXT_PUBLIC_TIDE_ELASTICSEARCH_HOST=https://a83890f7a31dea14e1ae83c6f0afacca.sdp2-b.elastic.sdp.vic.gov.au`
- - Create landing page with Custom collection component - See examples/nuxt-app/test/fixtures/landingpage/custom-collection.json props object for JSON config

### Checklist




#### For all PR's


#### For new components only
